script.js

```diff
@@ -0,0 +1,55 @@
+let library = localStorage.getItem('library') == null
+  ? []
+  : JSON.parse(localStorage.getItem('library'));
+
+function saveLocalStorage() {
+  localStorage.setItem('library', JSON.stringify(library));
+}
+
+function createTR(html) {
+  const div = document.createElement('table');
+  div.innerHTML = html;
+  return div.querySelector('tr');
+}
+
+function paintBooks() {
+  const bookList = document.getElementById('book-list');
+  bookList.innerHTML = '';
+  library.forEach((Book) => {
```

Comment:
- [ ] JavaScript follows the camelCase convention for naming variables, please use camel cases for your variable names. The variable you have named as `Book` should be `book`. You can find the best practices guide [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)